docs/reference/search/aggregations/reducer/derivative-aggregation.asciidoc

```diff
@@ -2,7 +2,7 @@
 === Derivative Aggregation
 
 A parent reducer aggregation which calculates the derivative of a specified metric in a parent histogram (or date_histogram) 
-aggregation. The specified metric must be numeric and the enclosing histogram must have `min_doc_count` set to `0`.
+aggregation. The specified metric must be numeric and the enclosing histogram must have `min_doc_count` set to `0` (default).
```

Comment:
Can we say `(default for histogram aggregation)` so its clear that the default refers to the min_doc_count value on the histogram and not on the validation of the reducer.